packages/components/scale-panel/src/ScalePanel.stories.tsx

```diff
@@ -0,0 +1,63 @@
+/** @jsx jsx */
+import React from 'react';
+import { jsx } from '@emotion/core';
+import { action } from '@storybook/addon-actions';
+import { number, select, withKnobs } from '@storybook/addon-knobs';
+import { storiesOf } from '@storybook/react';
+
+import { ScalePanel } from './ScalePanel';
+import { OrientationProps } from './ScalePanelInner';
+
+const stories = storiesOf('ui/ScalePanel', module).addParameters({
+  metadata: { author: 'CSSSR', status: 'Approved' },
+});
+
+interface StoryProps extends OrientationProps {
+  currentScale: number;
+  stepScale: number;
+  onChange(scale: number): void;
+  onChangeStep(step: number): void;
+}
+
+function useStoryProps(): StoryProps {
+  const [currentScale, setCurrentScale] = React.useState(100);
+  const [stepScaleMock, setStepScale] = React.useState(10);
+  const stepScale = number(
```

Comment:
```suggestion
  const stepScale = number(
    'stepScale',
    stepScaleMock,
    {
      min: 1,
      max: 100,
      range: true,
      step: 1,
    }
  );
```
Я бы не сдал здесь группировать knobs. 
Они все относятся к одному компоненту